### PR TITLE
zkapp update timing command  in test_zkapp_transaction (develop)

### PR DESCRIPTION
**Pull request by dkijania**
Explain your changes:
Added one more command to zkapp_test_transaction regarding update timings. Refactored code a bit so update permissions and timings use the same code.

Those changes where done during investigation of block slowness item

Explain how you tested your changes:
Run `local-network.sh` script with local changes regarding which zkapp command is run against nodes (I've reverted the code in my second commit, so one can check original code)  

Checklist:

- [ ] Dependency versions are unchanged
  - Notify Velocity team if dependencies must change in CI
- [ ] Modified the current draft of release notes with details on what is completed or incomplete within this project
- [ ] Document code purpose, how to use it
  - Mention expected invariants, implicit constraints
- [ ] Tests were added for the new behavior
  - Document test purpose, significance of failures
  - Test names should reflect their purpose
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them

### CODE REVIEW

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

File: scripts/mina-local-network/mina-local-network.sh [hunk 1]
```diff
@@ -764,7 +764,7 @@ if ${VALUE_TRANSFERS} || ${ZKAPP_TRANSACTIONS}; then
       let fee_payer_nonce++
       let sender_nonce++
 
-      QUERY=$(${ZKAPP_EXE} update-state --fee-payer-key ${FEE_PAYER_KEY_FILE} --nonce $fee_payer_nonce --zkapp-account-key ${ZKAPP_ACCOUNT_KEY_FILE} --zkapp-state $state --fee 5 | sed 1,5d)
+      QUERY=$(${ZKAPP_EXE} update-timings --fee-payer-key ${FEE_PAYER_KEY_FILE} --nonce $fee_payer_nonce --zkapp-account-key ${ZKAPP_ACCOUNT_KEY_FILE} --initial-minimum-balance 1 --cliff-time 0 --cliff-amount 0 --vesting-period 0 --vesting-increment 0 --fee 5 --current-auth Signature | sed 1,5d)
```
Comment (deepthiskumar) [inline scripts/mina-local-network/mina-local-network.sh]: Don't we want `update-timings` in addition to `update-state`?

**Review by deepthiskumar — APPROVED** [review]
Left a comment but otherwise looks good

File: scripts/mina-local-network/mina-local-network.sh (hunk 1 above)
Comment (dkijania) [reply scripts/mina-local-network/mina-local-network.sh]: I was thinking about that but finally  update-state is more general operation than timings. So didn't want to do this change without knowing more details. I'm planning to add better solution in the future where user can choose which command to run or mix

**Pull request merged**